pom.xml

```diff
@@ -58,7 +58,7 @@
     <version.commons.io>2.4</version.commons.io>
     <version.com.github.dblock>1.5</version.com.github.dblock>
     <version.com.samskivert.jmustache>1.11</version.com.samskivert.jmustache>
-    <version.com.squareup.okhttp>2.4.0</version.com.squareup.okhttp>
+    <version.com.squareup.okhttp>2.7.5</version.com.squareup.okhttp>
```

Comment:
this is now being managed by the parent pom - see https://github.com/hawkular/hawkular-parent-pom/pull/79
so we should upgrade the parent pom so we can pull that in and not have to manage this version ourselves.